components/homme/src/theta-l/share/viscosity_theta.F90

```diff
@@ -77,8 +79,7 @@ subroutine biharmonic_wk_theta(elem,stens,vtens,deriv,edgebuf,hybrid,nt,nets,net
 
    !if tensor hyperviscosity with tensor V is used, then biharmonic operator is (\grad\cdot V\grad) (\grad \cdot \grad) 
    !so tensor is only used on second call to laplace_sphere_wk
-   var_coef1 = .true.
-   if(hypervis_scaling > 0)    var_coef1 = .false.
+   var_coef1 = .false.
```

Comment:
Would this change, setting var_coef1 to false, cause issues with ne30 runs? This is not a compy issue.

Reply:
this is correct. for constant coefficient HV, var_coef is always .false.  for the tensor, it's false on the first call and true on the second cal.